src/migratus/database.clj

```diff
@@ -149,6 +149,43 @@
       (catch SQLException _
         false))))
 
+
+
+(defn migration-table-up-to-date?
+  [db table-name]
+  (sql/with-db-transaction
+    [t-con db]
+    (try
+      (sql/query t-con [(str "SELECT applied,description FROM " table-name)])
+      true
+      (catch SQLException _
+        false))))
+
+
+(defn create-migration-table!
+  "Creates the schema for the migration table via t-con in db in table-name"
+  [db modify-sql-fn table-name]
+  (log/info "creating migration table" (str "'" table-name "'"))
+  (sql/with-db-transaction
+    [t-con db]
+    (sql/db-do-commands t-con
+                        (modify-sql-fn
+                         (sql/create-table-ddl table-name [[:id "BIGINT" "UNIQUE" "NOT NULL"]
+                                                           [:applied "TIMESTAMP" "" ""]
+                                                           [:description "VARCHAR(1024)" "" ""]])))))
+
+(defn update-migration-table!
+  "Updates the schema for the migration table via t-con in db in table-name"
+  [t-con db modify-sql-fn table-name]
+  (log/info "updating migration table" (str "'" table-name "'"))
+  (sql/with-db-transaction
+    [t-con db]
+    (sql/db-do-commands t-con
+                        (modify-sql-fn
+                         [(str "ALTER TABLE " table-name " ADD COLUMN description varchar(1024)")
+                          (str "ALTER TABLE " table-name " ADD COLUMN applied datetime")]))))
```

Comment:
I think this should be timestamp, turns out datetime is not supported by Postgres as per this issue https://github.com/yogthos/migratus/issues/109